lib/ansible/modules/remote_management/redfish/redfish_command.py

```diff
@@ -296,7 +349,10 @@ def main():
     user = {'account_id': module.params['id'],
             'account_username': module.params['new_username'],
             'account_password': module.params['new_password'],
-            'account_roleid': module.params['roleid']}
+            'account_roleid': module.params['roleid'],
+            'account_updatename': module.params['update_username'],
+            'account_policy_name': module.params['account_policy_name'],
+            'account_policy_value': module.params['account_policy_value']}
```

Comment:
See previous naming comment. Suggest something like:
```
        'account_properties': module.params['account_properties']
```

Reply:
Suggestion has been adopted in code.